doc/using.rst

```diff
@@ -1,128 +1,111 @@
 .. _using-binder:
 
-============
-Using Binder
-============
+===============================
+Common usage patterns in Binder
+===============================
 
-This page describes how to use Binder to create interactive, sharable
-repositories. It includes how to prepare your repository to be binder-ready,
-as well as how to build and launch that repository with Binder.
+This page describes some common patterns and use-cases for Binder.
+If you're new to Binder, we recommend checking out :ref:`introduction`
+for an introduction to Binder, repository structure, and how to build your
+own repository.
 
-.. _preparing_repositories:
-
-Preparing a repository for Binder
-=================================
-
-All of the code in a repository is available to users,
-your Binder repository should contain at least:
-
-* The code you want users to run. This might be a collection of Jupyter
-  notebooks or scripts.
-* Configuration files (one or more text files) that specify the requirements
-  for building your project's code. For a complete list of supported files,
-  see :ref:`config-files`. Configuration files may be placed in the root of
-  your repository or in a ``binder/`` folder in the repository's root
-  (i.e. ``myproject/binder/``). If a ``binder/`` folder is used, Binder will
-  only read configuration files from that location and will ignore those in
-  the repository's root.
-
-.. tip::
+For a more complete list of sample repositories for use with Binder, see the
```

Comment:
Here, the word "Binder" is used as a service that connects to a repository (which is clear to me - it is just the use of Binder as it refers to a repo that messes with my head)

Reply:
Good point - we have always had a hard time disentangling these terms. Maybe something to put to rest at the dev meeting this year. In this PR I'll start using "Binder" to mean either "a service" or "the project", and I'll use binder-ready repository or binder repository for people's own work. That work?

Reply:
"binder-ready repository" works for me. I find "Binder repository" potentially confusing, as it could easily be read as "Binder project repository" (e.g. it implies ownership by the Binder project, whereas "binder-ready" means it is ready to be connected to the Binder service)